src/test/skript/tests/syntaxes/expressions/ExprBlockData.sk

```diff
@@ -1,4 +1,4 @@
-test "block data" when running minecraft "1.15.2":
+test "block data" when running minecraft "1.16.5":
```

Comment:
Shouldn't this be 1.13+ ?

Reply:
Didn't exist when I was testing, which is why I bumped the version up, because the test actually works now apparently.

Reply:
That's weird! I am afraid the test will not work on 1.13+ while the feature does so that might make the test not useful on those versions but if you don't have time to test it more it's okay we can leave it for later